tests/test_ab2cb.py

```diff
@@ -501,25 +522,25 @@ def test_neq(self):
         assert a != b
 
 
-@pytest.mark.parametrize('abcb', ad_urls)
-class TestAdURLs(object):
-    def test_url(self, abcb):
-        out = abcb_from_text(abcb.ab)
-        assert out == abcb
+# @pytest.mark.parametrize('abcb', ad_urls)
+# class TestAdURLs(object):
+#     def test_url(self, abcb):
+#         out = abcb_from_text(abcb.ab)
+#         assert out == abcb
```

Comment:
For what it's worth, the generated output from these tests looks correct to me; I think it's safe to include these tests and update the expected output to match.

Reply:
Updated tests to match what is output with https://github.com/brave/ab2cb/pull/5/commits/87cdb36bdf62932378a42f925b67842b6b52553c 😄 (also updated README after what you had ran into)